pkgs/applications/video/shaka-packager/default.nix

```
@@ -1,62 +1,59 @@
 { lib
 , stdenv
-, fetchurl
+, fetchFromGitHub
 , runCommand
 , shaka-packager
+, cmake
+, ninja
+, python3
+, darwin
+, nix-update-script
 }:
 
-let
-  sources = {
-    "x86_64-linux" = {
-      filename = "packager-linux-x64";
-      hash = "sha256-MoMX6PEtvPmloXJwRpnC2lHlT+tozsV4dmbCqweyyI0=";
-    };
-    aarch64-linux = {
-      filename = "packager-linux-arm64";
-      hash = "sha256-6+7SfnwVRsqFwI7/1F7yqVtkJVIoOFUmhoGU3P6gdQ0=";
-    };
-    x86_64-darwin = {
-      filename = "packager-osx-x64";
-      hash = "sha256-fFBtOp/Zb37LP7TWAEB0yp0xM88cMT9QS59EwW4MrAY=";
-    };
-  };
-
-  source = sources."${stdenv.hostPlatform.system}"
-    or (throw "Unsupported system: ${stdenv.hostPlatform.system}");
-in
 stdenv.mkDerivation (finalAttrs: {
   pname = "shaka-packager";
-  version = "2.6.1";
-
-  src = fetchurl {
-    url = "https://github.com/shaka-project/shaka-packager/releases/download/v${finalAttrs.version}/${source.filename}";
-    inherit (source) hash;
+  version = "3.0.4";
+
+  src = fetchFromGitHub {
+    owner = "shaka-project";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "shaka-packager";
```
that's wrong, the value is always this